examples/sandbox/index.ts

```diff
@@ -1044,21 +1041,18 @@ const $fioResults = $("#fioResults");
 $fioAddr.on("click", async (e) => {
   e.preventDefault();
   if (!wallet) {
-    $ethResults.val("No wallet?");
+    $fioResults.val("No wallet?");
     return;
   }
   if (core.supportsFio(wallet)) {
     const { addressNList } = wallet.fioGetAccountPaths({ accountIdx: 0 })[0];
-    let result = await wallet.fioGetPublicKey({
+    let result = await wallet.fioGetAddress({
```

Comment:
The TypeScript errors informed me that the fioGetPublicKey method does not exists and it does not